### DISCUSSION TITLE
Alteryx and Tableau

### DISCUSSION THREAD

**Discussion by FXADev** [f1db/f1db#19] [Q&A] [answered]
Is it possible to connect the data to Alteryx or Tabelau? 

Not sure how to do that.

**Comment by marceloverdijk** [comment]
I'm not familiar with Alteryx or Tabelau myself, but if anyone is interested I would say have a go at it.
This project provides multiple data formats (json, csv, sqlite); so if either Alteryx or Tabelau can use these formats I think it should be no problem.

**Reply by FXADev** [reply]
How do I get the CSV?

**Reply by marceloverdijk** [reply, marked as answer]
You can download them from the Releases page: https://github.com/f1db/f1db/releases

**Reply by Schadrac-Alcine** [reply]
If im not mistaken, Alteryx has a Tableau connector tool.

**Reply by Axsta** [reply]
would you overwrite your input file? If yes then automate the output to overwrite tableau input file. Or else you can append the data to a database and setup input connection for tableau to read the data.
